frontend/src/components/RecordsList.tsx

```diff
@@ -45,12 +45,13 @@ interface IRecordsListProps {
   setUserSearchVal: Dispatch<SetStateAction<string>>;
   showDownloadModal: boolean;
   setShowDownloadModal: Dispatch<SetStateAction<boolean>>;
-  handleDownload: (recordCount: number) => void;
+  handleDownload: () => void;
   samplesColDefs: ColDef[];
   sampleContexts?: DashboardSamplesQueryVariables["contexts"];
   userEmail?: string | null;
   setUserEmail?: Dispatch<SetStateAction<string | null>>;
   customToolbarUI?: JSX.Element;
+  exportDropdownItems?: IExportDropdownItem[];
```

Comment:
This is a new prop to `RecordsList`, but not a new prop to SMILE. `SamplesList` has been using this prop for "Export in Phenotype format for dbGaP".

This prop's type [`IExportDropdownItem`](https://github.com/mskcc/smile-dashboard/pull/231/files#diff-db966dd79103f32fddb49dcefcfba78d4325506ce364cd0e24779a3521aa7173R1238), however, has been changed to accept a custom export function to handle downloading from Databricks.